BSP/STM32F413/Src/BSP_Contactor.c

```diff
@@ -22,7 +22,7 @@ void BSP_Contactor_Init(void) {
 	GPIO_InitStruct.GPIO_Mode = GPIO_Mode_OUT;
 	GPIO_InitStruct.GPIO_Speed = GPIO_Speed_50MHz;
 	GPIO_InitStruct.GPIO_OType = GPIO_OType_PP;
-	GPIO_InitStruct.GPIO_PuPd = GPIO_PuPd_DOWN;
+	GPIO_InitStruct.GPIO_PuPd = GPIO_PuPd_UP;
```

Comment:
Why does the logic of the Contactor change now that we are checking if it is open or not?

Reply:
This was wrong before. It was correct on the master branch, but wrong on the rtos branch, and I picked the wrong one when I resolved the merge conflict.

This pin is connected to one of the contactor sense pins. The other contactor sense pin is connected to ground. When the contactors are closed, the contactors will short the contactor sense pins together, which ties this pin to ground externally. When the contactors are open, this pin will be floating externally, so we need the internal pull-up resistor to make this pin high when the contactors are open